### PR TITLE
gcp-oidc: manage API token in GCP Secret Manager

### PR DESCRIPTION
**Pull request by lunwang-ttd**

### CODE REVIEW

**Review by yishi-ttd — APPROVED** [review]

File: scripts/gcp-oidc/README.md [hunk 1]
```diff
@@ -104,32 +110,49 @@ Go to Admin portal [Enclave Id Management page](https://admin-integ.uidapi.com/a
...
+
 ### (For partner) Create VM Instance 
 There are a few placeholders that you need to replace in below command:
  - `{INSTANCE_NAME}`: your VM name, can be changed as your need.
+ - `{ZONE}`: which Google Cloud zone will be deployed on.
  - `{SERVICE_ACCOUNT}`: in `{SERVICE_ACCOUNT_NAME}@{PROJECT_ID}.iam.gserviceaccount.com` format, the one you created 
 in Prerequisites phase.
  - `{IMAGE_SHA}`: a valid UID2 operator image digest. You should have received this from UID2 team.
- - `{API_TOKEN}`: private operator api token, dedicated for you. You should have received this from UID2 team.
+ - `{API_TOKEN_SECRET_NAME}`: the secret name of your operator API token, the format is
```
Comment (yishi-ttd) [inline scripts/gcp-oidc/README.md:133]: Add a link to above Create secret section.

> the secret name of your operator API token created in above section[link]

Comment (lunwang-ttd) [reply scripts/gcp-oidc/README.md]: Added

File: scripts/gcp-oidc/README.md [hunk 2]
```diff
@@ -104,32 +110,49 @@ Go to Admin portal [Enclave Id Management page](https://admin-integ.uidapi.com/a
    - Protocol: "gcp-oidc"
    - Enclave ID: the generated value in Step 1
 
+### (For partner) Create secret of your private operator API token in Secret Manager
+Store your private operator API token provided by the UID2 team to Secret Manager and get the secret name which will be used to replace the `{API_TOKEN_SECRET_NAME}` placeholder later during VM instance creation.
+
+For example, following script creates a new secret `uid2_operator_api_token`, and prints secret name something like `projects/111111111111/secrets/uid2_operator_api_token/versions/1` which will be used to replace the `{API_TOKEN_SECRET_NAME}` placeholder later.
+```
+API_TOKEN="<YOUR_OPERATOR_API_TOKEN>"
```
Comment (yishi-ttd) [inline scripts/gcp-oidc/README.md:118]: Let's use {API_TOKEN} here to be consistent.

Also add below in the beginning of this small section:

`{API_TOKEN}`: private operator api token, dedicated for you. You should have received this from UID2 team.
In this section, we will guide you how to store `{API_TOKEN}` in GCP Secret Manager and get the secret name which will be used to replace the `{API_TOKEN_SECRET_NAME}` placeholder later during VM instance creation.

Comment (lunwang-ttd) [reply scripts/gcp-oidc/README.md]: Updated

**Pull request merged**